flybywiresim/a32nx
# feat: custom FCU font

## Summary of Changes

* Added custom FCU font
    * ~The new font is only for the numbers, not the labels above them~

## Screenshots (if necessary)



![image](https://user-images.githubusercontent.com/16512508/107893227-d3320d80-6ede-11eb-84c6-4f823d8c4dba.png)


## References

https://cdn.discordapp.com/attachments/758126266859388948/810248599946002462/P1020360.JPG
https://cdn.discordapp.com/attachments/758126266859388948/810248612147101706/P1020359.JPG
https://cdn.discordapp.com/attachments/758126266859388948/810248615343161354/P1020364.JPG




## Additional context





## Testing instructions

* Make sure the new font is accurate
* Make sure the decimal points behave as expected


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
